### PR TITLE
Test correct number of buckets in NumericHistogramAggregation

### PR DESCRIPTION
**Pull request by petroav**
* Add a test to ensure that the NumericHistogramAggregation function
throws an exception if it is given a number of buckets less than 1.

Testing: ran test in isolation and clean build

### CODE REVIEW

**Comment by petroav** [comment]
This commit tests the changes made by @cberner in https://github.com/facebook/presto/commit/461f0753b4d9bc55dfb0fb481cb98071fd4131c3.

@mattsfuller when you get a chance could you kindly review.

**Comment by mattsfuller** [comment]
Looks Good

File: presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestNumericHistogramAggregation.java [hunk 1]
```diff
@@ -40,6 +41,7 @@
 {
     private final AccumulatorFactory factory;
     private final Page input;
+    private final Page zeroBucketsInput;
```
Comment (martint) [inline presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestNumericHistogramAggregation.java]: I'd just inline this variable in the single place it's used

**Comment by petroav** [comment]
@martint I uploaded a new commit addressing your comment. I also rebased on master and ran the unit test again and everything passed. Thanks for taking a look!

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**